### ISSUE TITLE
performance-tweaks 0.0.1.r6.g2211975-1 syntax error on Pacman upgrade

### ISSUE THREAD

**Issue by FGD-Garuda** [chaotic-aur/packages#430]
Hello,

When upgrading to performance-tweaks 0.0.1.r6.g2211975-1, whether from Pamac or Fish, I always get that error

```
Reinstalling performance-tweaks (0.0.1.r6.g2211975-1)...
/tmp/alpm_cr7q4F/.INSTALL: line 11: syntax error near unexpected token `}'
Error while configuring performance-tweaks
/tmp/alpm_cr7q4F/.INSTALL: line 11: `}'
/usr/bin/bash: line 1: post_upgrade: command not found
Error: performance-tweaks: command failed to execute correctly
```

Whether I reinstall the package like above or issue an upgrade from a previous version like r4 or r5, the same syntax error appears.

Pacman/Pamac seem to tell me r6 is installed but I have no way to tell it is installed properly and functions properly.

I was unable to catch the makefile or build file or something during the upgrade and look at "line 11".

I tried to downgrade package and I get this
```

╰─λ sudo downgrade performance-tweaks
Available packages:

-  1)  performance-tweaks    0.0.1.r3.ga631469  1  any  (local)
-  2)  performance-tweaks    0.0.1.r4.gc494a03  1  any  (local)
-  3)  performance-tweaks    0.0.1.r5.ge17ef26  1  any  (local)
+  4)  performance-tweaks    0.0.1.r6.g2211975  1  any  (local)

select a package by number: 3
loading packages...
warning: downgrading package performance-tweaks (0.0.1.r6.g2211975-1 => 0.0.1.r5.ge17ef26-1)
resolving dependencies...
looking for conflicting packages...

Package (1)         Old Version          New Version          Net Change

performance-tweaks  0.0.1.r6.g2211975-1  0.0.1.r5.ge17ef26-1    0.00 MiB

Total Installed Size:  0.00 MiB
Net Upgrade Size:      0.00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                                                                                    [------------------------------------------------------------------------------------------] 100%
(1/1) checking package integrity                                                                                                                  [------------------------------------------------------------------------------------------] 100%
(1/1) loading package files                                                                                                                       [------------------------------------------------------------------------------------------] 100%
(1/1) checking for file conflicts                                                                                                                 [------------------------------------------------------------------------------------------] 100%
error: failed to commit transaction (conflicting files)
performance-tweaks: /etc/systemd/swap.conf.d/swap.conf exists in filesystem (owned by garuda-common-settings)
performance-tweaks: /etc/udev/rules.d/50-sata.rules exists in filesystem (owned by garuda-common-settings)
performance-tweaks: /etc/udev/rules.d/60-ioschedulers.rules exists in filesystem (owned by garuda-common-settings)
Errors occurred, no packages were upgraded.

```

I could not find information about this in forums, google, arch, or elsewhere.

I do not know what else to look for to provide more information if this is not reproducible on your end.

**Comment by FGD-Garuda** [comment]
Nevermind, just came across this

https://forum.garudalinux.org/t/upgrading-performance-tweaks/4131

Apparently a fix in the work, so I'll wait.

**Comment by dr460nf1r3** [comment]
Has this been fixed for you now?

**Comment by FGD-Garuda** [comment]
I think it has. The reason why I say "think" is that yesterday I did the upgrade on 1 snapshot and 2 VMs and it was working.

I continued the process today on my other snapshots and VMs but I am getting into something different

```
error: performance-tweaks: signature from "Pedro Henrique Lara Campos <root@pedrohlc.com>" is invalid
:: File /var/cache/pacman/pkg/performance-tweaks-0.0.1.r6.g2211975-1-any.pkg.tar.zst is corrupted (invalid or corrupted package (PGP signature)).
```
Also that package `librewolf-85.0.1-1-x86_64.pkg.tar.zst` fails to retrieve because `The requested URL returned error: 404`
Changed mirrors, changed Countries, refreshed 3 times using 2 ways, same result.

There's probably something going on with Chaotic-Aur, I guess. I'll try a few more things and see if I am stupid or if something really is going with these packages.


However to the specific coding error of performance-tweaks, since it did update fine 3 times **I am 100% confident it is fixed, yes.**
